split/helper.py

```diff
@@ -0,0 +1,138 @@
+import simplejson as json
+from split.models import Expense
+from split.models import Balance
+from split.models import Transaction
+
+from django.contrib.auth.models import User
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+"""
```

Comment:
Docstrings should be inside the function. Check any python open source code (could be django's) for reference